src/lib/components/banner.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import { closedBannerId, close } from '$lib/stores/banner';
+  import { cluster } from '$lib/stores/cluster';
+
+  const severities = {
+    High: 'high',
+    Medium: 'medium',
+    Low: 'low',
+  };
+
+  $: recommended = $cluster?.versionInfo?.recommended;
+  $: current = $cluster?.versionInfo?.current;
+  $: alert = $cluster?.versionInfo?.alerts?.[0];
+  $: severity = alert ? severities[alert.severity] : severities.Low;
+  $: show = recommended?.version && recommended.version != $closedBannerId;
+  $: message =
+    severity == severities.Low
+      ? `📥 v${recommended?.version} version is available`
+      : alert?.message;
+</script>
+
+{#if show}
+  <section class={`block leading-10 text-center ${severity}`}>
+    <a
+      href={`https://github.com/temporalio/temporal/releases/tag/v${$cluster.versionInfo.recommended.version}`}
```

Comment:
Svelte lets you interpolate using just curly braces in double-quoted strings. So, this could be.

```html
<a href="https://github.com/temporalio/temporal/releases/tag/v{$cluster.versionInfo.recommended.version}" target="_blank">
```

Reply:
addressed